### ISSUE TITLE
Linking to other identifiers does not work after "``foobar``" is introduced

### ISSUE THREAD

**Issue by brameret** [pdoc3/pdoc#106]
### Expected Behavior

When a docstring has a `` `module.identifier` ``, it should always resolve the link.

### Actual Behavior

When a docstring has a mention with double back quotes ` ``something`` `, afterwards the `` `module.identifier` `` does not resolve anymore.

### Steps to Reproduce

1. Create a `foobar/__init__.py` with the following content:
   ```python
   VERSION = "0"
   """
   The identifier `foobar.VERSION` links correctly to the version,
   but ``if there are double back quotes``,
   this link does not work anymore `foobar.VERSION`
   (without warning)
   """
   ```
2. `pdoc3 foobar --html`
3. See `html/index.html` that the first `foobar.VERSION` is resolved but not the second one.

### Additional info

- pdoc version: 0.6.3

**Comment by kernc** [comment]
If you just need a backtick literal, have you tried \\-escaping?

> To include a literal backtick character within a code span, you can use multiple backticks as the opening and closing delimiters:
>
>> ``There is a literal backtick (`) here.``

One of the code-parsing regular expressions seems to fail to account for this special case.

**Comment by brameret** [comment]
Thanks for your answer.

In fact, I did not use double backticks to introduce literal backtick. The double backticks just occurred in my docstrings, and I was very confused because I did not understand why "Linking to other identifiers" did not work with my project. I was just struggling with "fully qualified names" trying to make it work. It was one of the features that made me choose `pdoc3` :wink:
